var/spack/repos/builtin/packages/py-fenics-fiat/package.py

```diff
@@ -17,18 +17,20 @@ class PyFenicsFiat(PythonPackage):
     nonconforming elements"""
 
     homepage = "https://fenicsproject.org/"
-    url = "https://github.com/FEniCS/fiat/archive/2019.1.0.tar.gz"
-    git = "https://github.com/FEniCS/fiat.git"
+    url = "https://bitbucket.org/fenics-project/fiat/downloads/fiat-2019.1.0.tar.gz"
+    git = "https://bitbucket.org/fenics-project/fiat.git"
```

Comment:
According to https://bitbucket.org/fenics-project/fiat/src/master/, the project has moved to GitHub, so the previous url/git should be used instead.

Reply:
The problem with the GitHub Repo was, that it only includes the FIAT version starting from 2017.1.0, when I run the 'spack checksum' command. However to add the 2016.2.0 version (for consistency with the other packages), I suggest to use the tag from the fiat git Repo.
Note that this change is outsourced to a separate PR as well.